package.json

```diff
@@ -7,7 +7,7 @@
     "test": "mocha --reporter spec",
     "pretest": "npm run lint",
     "lint": "standard",
-    "prepare": "npm install require-self && require-self",
```

Comment:
Why ? Removing that means it's required to do npm install before doing npm publish

Reply:
Then we can change `prepare` to `prepublish`. `prepare` is also called on `npm install` with no arguments so it's installed 2 times as it's already in `devDependencies`.

Reply:
Even in [usage](https://github.com/yortus/require-self#usage) it's like that

Reply:
I believe prepublish is deprecated. If you find a way to make it run only on publish it would be perfect though

Reply:
But is it required to install it every time? It's not specified in the usage of require-self

Reply:
Every time what ?
It's required to install it if you want to run it.
If it's put in prepare it makes it possible to run npm publish on a clean copy without having to run npm install.

Reply:
Ideally, I would not want to install it not to run it in case of npm publish.
I only want to run it in case of a local install, not before publishing, but I believe that's not possible ?

Reply:
I'll put it back then